From 775a844b9bdb4db27731a642be12e1e4bc2276dd Mon Sep 17 00:00:00 2001
From: "Zhang, Qi" <qi1.zhang@intel.com>
Date: Tue, 27 Nov 2018 13:19:07 +0800
Subject: [PATCH 1320/1494] trusty: disable va printing in dmesg on user build

The va of trusty memory printing may leak security info

Change-Id: I7fafd1427b10e3bca48368a1e1327bbea3b2505c
Signed-off-by: Zhang, Qi <qi1.zhang@intel.com>
---
 drivers/trusty/trusty-virtio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/trusty/trusty-virtio.c b/drivers/trusty/trusty-virtio.c
index df066dd..113dc2f 100644
--- a/drivers/trusty/trusty-virtio.c
+++ b/drivers/trusty/trusty-virtio.c
@@ -335,7 +335,7 @@ static struct virtqueue *_find_vq(struct virtio_device *vdev,
 	 */
 	tvr->vr_descr->pa = (u32)HIULINT(pa);
 
-	dev_info(&vdev->dev, "vring%d: va(pa)  %p(%llx) qsz %d notifyid %d\n",
+	dev_dbg(&vdev->dev, "vring%d: va(pa)  %p(%llx) qsz %d notifyid %d\n",
 		 id, tvr->vaddr, (u64)tvr->paddr, tvr->elem_num, tvr->notifyid);
 
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 14, 0)
-- 
2.7.4

